CONTRIBUTING.md

```diff
@@ -150,6 +149,35 @@ You now have a Spark dataframe - try some commands out on it:
 Check out the [PySpark docs](https://spark.apache.org/docs/latest/api/python/getting_started/quickstart_df.html) for 
 more commands you can try out. 
 
+# Testing against a local Spark cluster
+
+When you run PySpark, it will create its own Spark cluster. If you'd like to try against a separate Spark cluster
+that still runs on your local machine, perform the following steps:
+
+1. Use [sdkman to install Spark](https://sdkman.io/sdks#spark). Run `sdk install spark 3.4.1` since we are currently
+building against Spark 3.4.1.
+2. `cd ~/.sdkman/candidates/spark/current/sbin`, which is where sdkman will install Spark.
+3. Run `start-master.sh` to start a master Spark node.
```

Comment:
this should be `./start-master.sh`

Reply:
I got away with "start-master.sh" as it was already on my PATH.